### ISSUE TITLE
Add ability to assign FS label in parted plugin

### ISSUE THREAD

**Issue by ykharuzhy** [gitbls/sdm#217]
It would be great if we can set a FS label when creating partitions with parted plugin.

I did some PoC here (checked for ext4 and should work for FAT): https://github.com/ykharuzhy/sdm/commit/611edfc0ea613c3ffbfea925bec9126e52ebb7a0 

Example: 
```
sdm" --batch --burnfile "$out_image" --burn-plugin parted:"addpartition=1024,ext4,primary,overlayroot+128,ext4,primary,media" "$out_image.tmp"
```

**Comment by gitbls** [comment]
What would the value of this be to users?

**Comment by ykharuzhy** [comment]
I use this to make rootfs overlay backed by partition, not tmpfs. Mounting
by label allow to have one image for SD and NVMe.

пт, 14 чэр 2024, 19:09 карыстальнік Benn ***@***.***> напісаў:

**Comment by gitbls** [comment]
I'll have a look into this next week.

**Comment by gitbls** [comment]
This will be in the next sdm release. Thanks for the POC!

**Comment by gitbls** [comment]
This is included in V12.4. I'd appreciate it if you could test/verify. Thx!

**Comment by gitbls** [comment]
Closed due to no activity.
